staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -185,6 +192,11 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"it out. This is the default request timeout for requests but may be overridden by flags such as "+
 		"--min-request-timeout for specific types of requests.")
 
+	fs.DurationVar(&s.PostStartHookGracePeriod, "post-start-hook-grace-period", s.RequestTimeout, ""+
+		"An optional field indicating the duration that poststarthook liveness health checks will return true "+
```

Comment:
Better avoid liveness check as a term and use `/healthz` instead.

Reply:
Would not use poststarthook as a term. Better something like `Period after which /healthz returns failure if the apiserver does not become ready (reported via /readyz) on startup. Zero for immediate /healthz failure until it is ready.`.